package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sn-simple-task-editor",
-  "version": "1.3.5",
+  "version": "1.3.6",
```

Comment:
Could you revert this as well? Sorry should have mentioned this earlier. We handle versioning ourselves when we release it.

Reply:
sure, should have known that

Reply:
package-lock.json should be omitted completely from the PR as well.